e2e-playwright/dashboard-cujs/README.md

```diff
@@ -0,0 +1,35 @@
+# Dashboard Critical User Journeys (CUJs) E2E Tests
+
+This directory contains end-to-end tests for critical user journeys related to dashboard functionality in Grafana, specifically testing AdHoc Filters, GroupBy Variables, Scopes, and Dashboard Navigation. The test suite validates core dashboard workflows including filtering data across dashboards, autocomplete functionality, operator selection, grouping across multiple dimensions, scope selection and persistence, and navigation between dashboards with state management. Tests use three pre-configured dashboard JSON files (cuj-dashboard-1, cuj-dashboard-2, cuj-dashboard-3) that contain text panels displaying variable values, AdHoc filter variables, GroupBy variables, and time range controls connected to a Prometheus datasource.
+
+## Environment Flags
+
+### `NO_DASHBOARD_IMPORT`
+
+**Default**: `false`
+
+When set to `true`, skips importing test dashboards during global setup. Use this when running against a live Grafana instance that already has the required dashboards with matching UIDs (`cuj-dashboard-1`, `cuj-dashboard-2`, `cuj-dashboard-3`).
+
+```bash
+NO_DASHBOARD_IMPORT=true yarn e2e:playwright
+```
+
+### `API_CONFIG_PATH`
+
+**Default**: `../dashboards/cujs/config.json`
+
+Configures the path to the API mocking configuration file. This enables dynamic API endpoint configuration for different environments. The config file should contain endpoint patterns for labels and values APIs. When set, the test suite will make real API calls instead of using mocked responses.
```

Comment:
@mdvictor can you be more specific here about what labels and values APIs are? Let's just put info it's for resolving adhoc/group by dimenstions and values. Someone not experienced with grafana may be confused by what these are.

Reply:
Yes indeed, good point, thank you! Will make it clearer!